dev_utils/config.yaml

```
@@ -23,6 +23,7 @@ backup:
   cacert: "./dev_utils/certs/ca.pem"
   # posix backend
   location: "dev_utils"
+  header: "false"
```

```suggestion
  copyHeader: "false"
```
A more verbose name is preferable .

---

done